buildpacks/nodejs/lib/build.sh

```diff
@@ -16,6 +16,28 @@ source "$bp_dir/lib/utils/log.sh"
 # shellcheck source=/dev/null
 source "$bp_dir/lib/utils/toml.sh"
 
+clear_cache_on_stack_change() {
+	local layers_dir=$1
+
+	if [[ ! -f "${layers_dir}/store.toml" ]]; then
+		touch "${layers_dir}/store.toml"
+		cat <<TOML >"${layers_dir}/store.toml"
+[metadata]
+last_stack = "$CNB_STACK_ID"
+TOML
+	else
+		if [[ $CNB_STACK_ID != $(toml_get_key_from_metadata "${layers_dir}/store.toml" "last_stack") ]]; then
```

Comment:
`$(cat "${layers_dir}/store.toml" | grep "last_stack")` outputs `last_stack` and not `last_stack = "heroku-18"` like its supposed to